torchgeo/datamodules/eurosat.py

```diff
@@ -67,3 +67,59 @@ def __init__(
                 :class:`~torchgeo.datasets.EuroSAT`.
         """
         super().__init__(EuroSAT, batch_size, num_workers, **kwargs)
+
+
+class EuroSAT100DataModule(NonGeoDataModule):
+    """LightningDataModule implementation for the EuroSAT100 dataset.
+
```

Comment:
Can you add the "Intended for tutorials and demonstrations, not for benchmarking." disclaimer here too

Reply:
Done.